pkg/api/getambassador.io/v3alpha1/crd_listener.go

```diff
@@ -178,6 +190,7 @@ type ListenerSpec struct {
 // +kubebuilder:printcolumn:name="StatsPrefix",type=string,JSONPath=`.spec.statsPrefix`
 // +kubebuilder:printcolumn:name="Security",type=string,JSONPath=`.spec.securityModel`
 // +kubebuilder:printcolumn:name="L7Depth",type=string,JSONPath=`.spec.l7Depth`
+// +kubebuilder:printcolumn:name="HTTP3Options",type=string,JSONPath=`.spec.http3Options`
```

Comment:
1. Given the types in this PR, that syntax is wrong I think? I think it'd need to be `JSONPath=`.spec.http3Options.altSvc` in order to have `type=string`.

2. I'm thinking that's an awful lot to include in the default `kubectl get listeners` table. I do think the H/3 port would be good to include though. Assuming you take my suggestion of splitting `http3Options.altSvc` in to structured fields, I think it would be good to rename the "Port" column and then add an "H3 Port" next to it.
```go
// +kubebuilder:printcolumn:name="TCP/H1/H2 Port",type=string,JSONPath=`.spec.port`
// +kubebuilder:printcolumn:name="H3 Port",type=string,JSONPath=`.spec.http3Options.port`
```